### ISSUE TITLE
Does "instances_attributes_val2020" annotation has wrong ids?

### ISSUE THREAD

**Issue by janmejaya-nanda** [cvdfoundation/fashionpedia#5]
Annotation for "attributes" in "instances_attributes_val2020" json looks something like,
`'attributes': [{'id': 234, 'name': 'no opening', 'supercategory': 'opening type', 'level': 1, 'taxonomy_id': 'att000242_00'}, {'id': 281, 'name': 'plastic', 'supercategory': 'non-textile material type', 'level': 1, 'taxonomy_id': 'att000298_00'},........, {'id': 340, 'name': 'plant', 'supercategory': 'textile pattern', 'level': 1, 'taxonomy_id': 'att000363_00'}]`
there is a jump in ID from 234 to 281 and it ends on 340. 
**I am assuming there are some wrong annotations, correct me if I am wrong.**
if these annotations are correct, then how one can inference attribute from category? where to get relation between `category` and `attribute`?
In case of inferencing done through [fashionpedia detection](https://github.com/tensorflow/tpu/blob/master/models/official/detection/projects/fashionpedia/inference.py), How one can extract attribute values from the saved check point?

**Comment by KMnP** [comment]
Hi, thanks for the question.
These attributes ids, like COCO dataset, are not consecutive.  We initially had more attributes but deleted some due to lack of annotated instances or difficult to annotate. 

In the proposed detection model, we treat attribute and category classification as two separate tasks, thus the model cannot inference attribute directly from category.  But it can predict attributes given the image.

Please let me know if you have any other questions

**Comment by janmejaya-nanda** [comment]
Thanks for a quick reply. I have a couple of follow-up questions
1. [these checkpoints](https://github.com/tensorflow/tpu/tree/master/models/official/detection/projects/fashionpedia#checkpoint) provides attributes (as logits) of the shape (# category, 294). can I safely assume logits at index 235 belongs to `id=281`? 
2. As the training data(of Kaggel) suggests, one class in an image can have multiple attributes. So, there is some sort of thresholding needs to be done on top of attribute scores than the `argmax`?

**Comment by KMnP** [comment]
Hi, sorry for the late reply. Did you find the answers already?
